lib/diagnostic.rb

```diff
@@ -106,12 +128,22 @@ def eat_meat(food)
 
 ##
 # your response here
+class Lion < Cat
+  include Carnivorous
+end
+def roar
+  puts "ROAR!"
+end
 ##
```

Comment:
move that `end` on line 133 to the end. Make sure all methods and include are part of the Class!